app_dart/lib/src/request_handlers/reset_try_task.dart

```diff
@@ -26,13 +26,17 @@ class ResetTryTask extends ApiRequestHandler<Body> {
   static const String kOwnerParam = 'owner';
   static const String kRepoParam = 'repo';
   static const String kPullRequestNumberParam = 'pr';
+  static const String kBuilderParam = 'builders';
 
   @override
   Future<Body> get() async {
     checkRequiredQueryParameters(<String>[kRepoParam, kPullRequestNumberParam]);
     final String owner = request!.uri.queryParameters[kOwnerParam] ?? 'flutter';
     final String repo = request!.uri.queryParameters[kRepoParam]!;
     final String pr = request!.uri.queryParameters[kPullRequestNumberParam]!;
+    final String? builders = requestData![kBuilderParam] as String? ?? '';
+    // The [builders] parameter is expecting comma joined string, e.g. 'builder1, builder2'.
```

Comment:
does it support builders with white spaces?

Reply:
Yes. We retrieve the builders splitting based on comma. It supports builders either with space or not.